Content.Server/Nutrition/Components/HungerComponent.cs

```diff
@@ -22,49 +22,54 @@ public sealed class HungerComponent : SharedHungerComponent
     {
         [Dependency] private readonly IRobustRandom _random = default!;
 
+        [DataField("base_decay_rate")]
```

Comment:
This isn't your fault but I don't know why this isn't capitalized like baseDecayRate